pkg/k8s/service.go

```diff
@@ -76,6 +76,16 @@ func getAnnotationServiceForwardingMode(svc *slim_corev1.Service) loadbalancer.S
 
 	return loadbalancer.SVCForwardingModeSNAT
 }
+func getAnnotationServiceLoadBalancingAlgo(svc *slim_corev1.Service) loadbalancer.SVCLoadBalancingAlgo {
+	if value, ok := annotation.Get(svc, annotation.ServiceLoadBalancingAlgo); ok {
+		val := loadbalancer.ToSVCLoadBalancingAlgo(strings.ToLower(value))
+		if val != loadbalancer.SVCLoadBalancingAlgoUndef {
+			return val
+		}
```

Comment:
Definitely need to warn if an unknown algorithm is here!

Reply:
Agree, log.Warn or the like would be good indeed to indicate we're falling back to the default.

Reply:
done